src/game_renderer/GameRenderer.cpp

```diff
@@ -186,9 +187,33 @@ void GameRenderer::RenderIt(GLFWwindow* window,
   game_state->SetItemsInView(
       GameRenderer::GetObjectsInView(vfplane, level->getTree()));
 
+  // large far for sexy looks
+  P->popMatrix();
+  P->pushMatrix();
+  P->perspective(45.0f, aspect, 0.01f, 1000.0f);
+
+  // Ground
+  std::shared_ptr<Program> current_program;
+  std::shared_ptr<Texture> current_texture;
+  if (player->GetGround()) {
```

Comment:
It would probably be cleaner to implement this in the shader, but I don't care that much.